api/rpc/account/account.proto

```diff
@@ -146,7 +146,8 @@ message SignUpRequest {
 
 message VerificationRequest {
   string name = 1;
-  string code = 2;
+  string password = 2;
```

Comment:
As for string vs byte I don't see a way to specify a fixed length either way: https://developers.google.com/protocol-buffers/docs/proto3#scalar I would stick with string since our hashing returns a string.